src/datalakebundle/table/schema/MetadataChecker.py

```diff
@@ -0,0 +1,90 @@
+import json
+from logging import Logger
+from pyspark.sql.session import SparkSession
+from pyspark.sql.utils import AnalysisException
+
+from datalakebundle.table.create.TableDefinition import TableDefinition
+from deepdiff import DeepDiff
+
+
+class MetadataChecker:
+    def __init__(self, logger: Logger, spark: SparkSession):
+        self.__logger = logger
+        self.__spark = spark
+
+    def check(self, table_definition: TableDefinition):
+        tables = self.__spark.sql(f"SHOW TABLES IN {table_definition.db_name}").toPandas()
```

Comment:
Not sure but I expect that this would require pandas to be installed?

Reply:
`self.__spark.sql(f"SHOW TABLES IN {table_definition.db_name}").collect()`

Reply:
Fixed